docs/trace/getting-started/README.md

```diff
@@ -50,15 +50,42 @@ What does the above program do?
 
 The program creates an `ActivitySource` which represents an [OpenTelemetry
 Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracer).
+
+```csharp
+private static readonly ActivitySource MyActivitySource = new ActivitySource(
+        "MyCompany.MyProduct.MyLibrary");
+```
+
 The `ActivitySource` instance is used to start an `Activity` which represents an
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span).
+
+```csharp
+using (var activity = MyActivitySource.StartActivity("SayHello"))
+        {
+            activity?.SetTag("foo", 1);
+            activity?.SetTag("bar", "Hello, World!");
+            activity?.SetTag("baz", new int[] { 1, 2, 3 });
+        }
+```
+
 An OpenTelemetry
 [TracerProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracerprovider)
 is configured to subscribe to the activities from the source
 `MyCompany.MyProduct.MyLibrary`, and export it to `ConsoleExporter`.
 `ConsoleExporter` simply displays it on the console.
 
+```csharp
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .SetSampler(new AlwaysOnSampler())
+            .AddSource("MyCompany.MyProduct.MyLibrary")
+            .AddConsoleExporter()
+            .Build();
+```
+
+> [!NOTE]
+> If the user doesn't specify a sampler then a default samper is used. The above example explicitly sets the sampler to the `AlwaysOnSampler`.
```

Comment:
I'd suggest to not talk about sampling in a getting-started doc. (we should remove it from the code as well).
This is absolutely for getting started - the next doc (customizing..) talks about sampling and all other options.